Libraries/zephyr/MAX/CMakeLists.txt

```diff
@@ -60,3 +60,4 @@ add_subdirectory_ifdef(CONFIG_SOC_MAX32680 Source/MAX32680)
 add_subdirectory_ifdef(CONFIG_SOC_MAX32690 Source/MAX32690)
 add_subdirectory_ifdef(CONFIG_SOC_MAX78000 Source/MAX78000)
 add_subdirectory_ifdef(CONFIG_SOC_MAX78002 Source/MAX78002)
+add_subdirectory_ifdef(CONFIG_SOC_MAX32657 Source/MAX32657)
```

Comment:
Should this line be moved upper to make the order of MCUs alphanumeric?

Reply:
Thanks, all them addressed, 
I2C related case shall be handled while working on drivers.